### ISSUE TITLE
[ISSUE]No wifi calling with afwall enabled

### ISSUE THREAD

**Issue by dieharddan** [ukanth/afwall#1209]
Wifi calling quits working with afwall enabled; I tried selecting all the apps except (Any app) and it still doesn't work, if I select (Any app) then it will work

**Firewall Logs**
[IPv4rules.log](https://github.com/ukanth/afwall/files/6410719/IPv4rules.log)

**Smartphone (please complete the following information):**
 - Device: lg v40
 - Android OS: stock android 10 rooted unlocked bootloader

**Additional context**

**Comment by ukanth** [comment]
Please enable ipv6 from preference.

**Comment by dieharddan** [comment]
"ipv6 support" in preference is already enabled

**Comment by ukanth** [comment]
Yes. I can see it's enabled. Can you turn on log service and see if any blocked ports ?

**Comment by dieharddan** [comment]
Ok i enabled "turn on log service".
[IPv6rules.log](https://github.com/ukanth/afwall/files/6411307/IPv6rules.log)

**Comment by ukanth** [comment]
Are you not getting notifications on blocked apps ?

**Comment by dieharddan** [comment]
Well I use signal app, and dont receive texts while that is blocked

**Comment by MrEngineerMind** [comment]
Your issue is that there is no wifi calling when afwall enabled.

That means afwall is blocking "something" that wifi needs to work.

So, to figure out what that "something" is, do these steps:
1) Turn on the log service
2) Reboot your phone.
3) Try to use wifi calling
4) Now view the log - it will list all the modules that tried to access the internet, but were blocked by wifi.
5) Then post in this ticket that list of modules that were in the log. We can then try to figure out which module is causing the wifi calling to not work.

**Comment by dieharddan** [comment]
[iptables.log](https://github.com/ukanth/afwall/files/6412643/iptables.log)

**Comment by ukanth** [comment]
have you allowed "Root" from list of apps ?

**Comment by dieharddan** [comment]
Everything is enabled except root aps and all apps
Because I didnt want necessarily give all root aps internet access

**Comment by dieharddan** [comment]
Enabling root apps does seem to allow wifi calling again

**Comment by xcalc** [comment]
Allow the following:
* kernel (-11) - all networks
* vendor.qti.iwlan (10123) - all networks
and reboot.
In case exclude from rethink dns.
Just hit this in Android 16.
